mcc/odin/odin/snapshot_list_test.go

```diff
@@ -161,14 +161,29 @@ var listSnapshotsCases = []listSnapshotsCase{
 		},
 		instanceID: "",
 	},
+	// Instance selection
+	{
+		testCase: testCase{
+			expected: []*rds.DBSnapshot{
+				exampleSnapshot2,
+			},
+			expectedError: "",
+		},
+		name: "Two instances two snapshots",
+		snapshots: []*rds.DBSnapshot{
+			exampleSnapshot1,
+			exampleSnapshot2,
+		},
+		instanceID: "develop-rds",
+	},
```

Comment:
This is a new `listSnapshotCase` for the test case where a instance name/ID has been specified.